enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -413,7 +416,9 @@ func TestFetchRemoteBuildOutputs(t *testing.T) {
 		})
 		return outputPath, err
 	}
-	downloadedOutputPath, err := findFile(remotebazel.BuildBuddyArtifactDir, "hello_world_go")
+	err := os.Chdir(repoDir)
```

Comment:
nit: use `t.Chdir(repoDir)` everywhere to revert the directory change during test cleanup